corgi/collectors/pnc.py

```diff
@@ -33,7 +33,11 @@ def __init__(self, data: dict[str, Any]):
         # Temporary workaround for component purl conflicts
         self.components = {}
         created_nvrs = set()
-        for component in data["components"]:
+        # Skip the first entry, which is a duplicate of the root component
+        # Otherwise we will create an extra node, quarkus-bom will provide quarkus-bom
+        # so it will appear twice in our manifeats, and IDs will not be unique
+        # this breaks parsing of SPDX manifests by clients
+        for component in data["components"][1:]:
```

Comment:
Are we 100% sure that this will always be the first entry in the SBOMer manifest?

Reply:
The first component should, afaik, be the same as the "root" (viz. the metadata) component because it has to have a bomref in the components section to be referenced in the relationships section. Obviously, that _could_ change, but I think that would be an exceptional circumstance.